lib/streamlit/util.py

```diff
@@ -451,6 +451,32 @@ def print_url(title, url):
     click.secho(url, bold=True)
 
 
+def process_gitblob_url(url):
+    """Checks url to see if it describes a github gist "blob" URL.  
+    If so, returns a new URL to get the "raw" script. 
+    If not, returns URL unchanged.
+    """
+    # Matches github.com and gist.github.com.  Will not match githubusercontent.com.
+    # See this regex with explainer and sample text here: https://regexr.com/4odk3
+    re_github_or_gist = re.compile("(?P<base>https:\/\/?(gist.)?github.com\/)(?P<account>([\w\.]+\/){1,2})(?P<blob_or_raw>(blob|raw))?(?P<suffix>(.+)?)")
```

Comment:
Move this into a top-level const, and break into multiple lines for readability:

For example:

```py
GITHUB_RE = re.compile(
    "(?P<base>https:\/\/?(gist.)?github.com\/)"
    "(?P<account>([\w\.]+\/){1,2})"
    "(?P<blob_or_raw>(blob|raw))?"
    "(?P<suffix>(.+)?)")
```